### PR TITLE
v9.3 Changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

**Review by cpoile — APPROVED** [review]

File: source/deploy/mattermost-changelog.md [hunk 1]
```diff
@@ -15,6 +15,100 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
...
+ - Updated minimum required Firefox version to v115+.
+ - Updated minimum supported Chromium version to 118+.
+
+### Improvements
+
+#### User Interface (UI)
+ - Updated pre-packaged Playbooks plugin version to [v1.39.1](https://github.com/mattermost/mattermost-plugin-playbooks/releases/tag/v1.39.1).
+ - Updated pre-packaged Calls version [v0.21.1](https://github.com/mattermost/mattermost-plugin-calls/releases/tag/v0.21.1).
+ - Added Vietnamese (Beta) as a new language.
+ - Added support for custom keywords with highlights without a notification.
```
Comment (johndavidlugtu) [inline source/deploy/mattermost-changelog.md:35]: Added ability to passively track keywords with highlights without triggering a notification

**Review by johndavidlugtu — CHANGES_REQUESTED** [review]

**Review by agnivade — APPROVED** [review]

**Review by marianunez — APPROVED** [review]

File: source/deploy/mattermost-changelog.md [hunk 2]
```diff
@@ -15,6 +15,103 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
...
+ - Updated minimum supported Chromium version to 118+.
+
+### Improvements
+
+#### User Interface (UI)
+ - Updated pre-packaged Playbooks plugin version to [v1.39.1](https://github.com/mattermost/mattermost-plugin-playbooks/releases/tag/v1.39.1).
+ - Updated pre-packaged Calls version to [v0.21.1](https://github.com/mattermost/mattermost-plugin-calls/releases/tag/v0.21.1).
+ - Updated pre-packaged Jira plugin version to [v4.0.1](https://github.com/mattermost/mattermost-plugin-jira/releases/tag/v4.0.1). Also see [v4.0.0](https://github.com/mattermost/mattermost-plugin-jira/releases/tag/v4.0.0) for recent breaking changes.
+ - Added Vietnamese (Beta) as a new language.
+ - Added the ability to passively track keywords with highlights without triggering a notification.
```
Comment (johndavidlugtu) [inline source/deploy/mattermost-changelog.md:36]: Remind me, do we usually callout in the changelog on whether the feature is paid or not? @amyblais

File: source/deploy/mattermost-changelog.md [hunk 3]
```diff
@@ -15,6 +15,103 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
...
+#### Performance
+ - Removed pre-fetch preference and set new prefetch limits for the webapp.
+ - Improved websocket event marshaling performance.
+ - Batched loading of recently used emojis on initial load.
+
+#### Administration
+ - The tooltip on the announcement bar in the **System Console** is now widened.
+ - Improved the error message when trying to activate a plugin in an unsupported environment.
+ - Added a file storage permission check to the workspace health dashboard.
+ - Performed a cleanup in preparation for adding support for multi-word mentions.
```
Comment (johndavidlugtu) [inline source/deploy/mattermost-changelog.md:53]: Performed a cleanup in preparation for adding support for multi-word keywords that trigger notification.

File: source/deploy/mattermost-changelog.md [hunk 4]
```diff
@@ -15,6 +15,103 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
...
+ - Introduced trace logging level for LDAP messages.
+ - Added a new way to modify ``WebSocket`` messages sent to individual connections.
+ - Added a new server side hook ``MessagesWillBeConsumed`` to allow modifying post objects after they are grabbed from the database but before they are delivered to the client. This is behind a feature flag and disabled by default.
+ - Users and posts are now pretty-printed in the logs.
+ - Improved file extraction logging.
+ - Exposed ``ThreadView`` and ``AdvancedCreateComment`` components in the webapp plugin exported components list.
+ - Added **Logging > Advanced Logging** setting to the **System Console** to allow admins to configure custom log targets via the user interface.
+
+### Bug Fixes
+ - Fixed an issue where marking a Group Message as unread didn't show the mention badges correctly.
```
Comment (johndavidlugtu) [inline source/deploy/mattermost-changelog.md:67]: ... didn't show the badge count correctly.

**Review by johndavidlugtu — CHANGES_REQUESTED** [review]

**Review by gabrieljackson — APPROVED** [review]

File: source/deploy/mattermost-changelog.md (hunk 2 above)
Comment (amyblais) [reply source/deploy/mattermost-changelog.md:36]: @johndavidlugtu Yes, what should I add here?

Comment (johndavidlugtu) [reply source/deploy/mattermost-changelog.md:36]: A callout that it this feature is available for Professional and Enterprise customers only

**Review by johndavidlugtu — APPROVED** [review]

File: source/deploy/mattermost-changelog.md [hunk 5]
```diff
@@ -15,6 +15,106 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
...
+ - Improved websocket event marshaling performance.
+ - Batched loading of recently used emojis on initial load.
+
+#### Administration
+ - The tooltip on the announcement bar in the **System Console** is now widened.
+ - Improved the error message when trying to activate a plugin in an unsupported environment.
+ - Added a file storage permission check to the workspace health dashboard.
+ - Performed a cleanup in preparation for adding support for multi-word keywords that trigger notification.
+ - Added a warning log message when the app runs as root.
+ - Removed all uses of ``ExperimentalTimezone`` in webapp.
```
Comment (ThiefMaster) [inline source/deploy/mattermost-changelog.md:55]: When reading this as a mattermost user, I can't help but wonder if this means that the timezone feature was removed (extremely unlikely) or that it's now always enabled and no longer behind a config setting (much more likely).

Comment (amyblais) [reply source/deploy/mattermost-changelog.md:55]: Thank you, I'll clarify this in the changelog.

**Pull request merged**